CONTRIBUTING.md

```diff
@@ -50,14 +48,20 @@ charmcraft pack
 
 ### Deploy
 
-<!-- TEMPLATE-TODO: Update the deploy command for name of charm-->
-
 ```bash
+# Import container
+git clone https://github.com/canonical/pgbouncer-container.git
+# Build container locally, since it's not been exported anywhere yet.
+docker build . -t pgbouncer:0.8
+docker save pgbouncer:0.8 -o pgb.tar
+# Import container file into microk8s container registry
+microk8s ctr image import pgb.tar
+
 # Create a model
 juju add-model dev
 # Enable DEBUG logging
 juju model-config logging-config="<root>=INFO;unit=DEBUG"
 # Deploy the charm
 juju deploy ./template-operator_ubuntu-20.04-amd64.charm \
```

Comment:
This can be changed to `pgbouncer-k8s_ubuntu-20.04-amd64.charm`.

Reply:
Thanks for catching this! Updated.